apertium_apy/apy.py

```diff
@@ -178,11 +185,17 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    if(found is False):
```

Comment:
1) don't use parens in conditionals in Python
2) in any language, don't use `!= false` or equivalent, use `not`
3) in general, reverse if/else so that you never have to use `not`

Reply:
Thanks for telling that, I have changed the syntax.